### PR TITLE
Proposal: switch travis to U14 Trusty

### PR DESCRIPTION
**Pull request by dvoytenko**
Trusty version comes with some fresher libraries, including Phantom2. For more details see: 

http://docs.travis-ci.com/user/trusty-ci-environment/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 language: node_js
+sudo: required
```
Comment (cramforce) [inline .travis.yml]: Why?

Comment (dvoytenko) [inline .travis.yml]: Forgot to mention the link: http://docs.travis-ci.com/user/trusty-ci-environment/

**Comment by dvoytenko** [comment]
@cramforce the friendliest of pings.

File: .travis.yml (hunk 1 above)
Comment (cramforce) [inline .travis.yml]: Can you add this as a comment

**Comment by cramforce** [comment]
LGTM

**Pull request merged**